pages/seeker/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styles from './seeker.module.css'
+
+const lerp = (start: number, end: number, percent: number) => {
+   return start + (start - end) * percent
+}
+
+interface IFollowerProps {
+  children: React.ReactNode
+  followerRef: React.RefObject<HTMLDivElement>
+}
+
+const MouseFollower = (props: IFollowerProps) => {
+  const [followerLeft, setFollowerLeft] = React.useState(0)
+  const [followerTop, setFollowerTop] = React.useState(0)
+  const [mouseLeft, setMouseLeft] = React.useState(0)
+  const [mouseTop, setMouseTop] = React.useState(0)
+
+  React.useEffect(() => {
+    document.onmousemove = function(e) {
+      setMouseLeft(e.pageX)
+      setMouseTop(e.pageY)
+    }
+  })
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+        setFollowerTop(lerp(followerTop, mouseTop, 0.01))
+        setFollowerLeft(lerp(followerLeft, mouseLeft, 0.01))
+    }, 10)
+  }, [mouseLeft, mouseTop, followerTop, followerLeft])
```

Comment:
The follower position should really be a Ref instead of state. Or perhaps even get a ref of the follower div, so you don't need any state at all.